src/main/java/Duke.java

```diff
@@ -6,5 +8,77 @@ public static void main(String[] args) {
                 + "| |_| | |_| |   <  __/\n"
                 + "|____/ \\__,_|_|\\_\\___|\n";
         System.out.println("Hello from\n" + logo);
+
+        Task[] texts = new Task[100];
+        String line;
+        String secondline;
+        int NumberOfTexts = 0;
+        int TextNumber;
+        Scanner in = new Scanner(System.in);
+
+        System.out.println("_______________________________________________");
+        System.out.println("Hello! I'm Duke");
+        System.out.println("What can i do for you?");
+        System.out.println("\n_______________________________________________");
+
+        do{
+            line = in.next();
+            if(!line.equals("bye") && !line.equals("list") && !line.equals("done") && !line.equals("todo")
```

Comment:
I think this condition might be too long and may be hard to read.
I think it is possible to put it at the else part of a if-else clause so that you don't need to write this long condition out.